packages/bathymetry/src/index.ts

```diff
@@ -53,6 +62,16 @@ class CreateAction extends CommandLineAction {
                 process.env[Env.Gdal.DockerContainerTag] = 'ubuntu-full-latest';
             }
         }
+        const tileMatrixInput = this.tileMatrix.value ?? GoogleTms.identifier;
+        const tileMatrix = TileMatrixSets.find(tileMatrixInput);
+        if (tileMatrix == null) {
```

Comment:
Do we still need checking here since the default is already `GoogleTms.identifier`

Reply:
`.value`  is `string | undefined` so typescript wouldn't let me pass it to the `.find()` method